### PR TITLE
Update enabling-multiple-custom-tracking-domains.md

### PR DESCRIPTION
**Pull request by azikh**

### CODE REVIEW

File: articles/tech-resources/enabling-multiple-custom-tracking-domains.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ To configure one or more custom tracking domains through the SparkPost user inte
     ![](media/enabling-multiple-custom-tracking-domains/xU06kJ2lac6YGEIi8KlReq8OaF1kfX.png)
 1. Complete the Sending Domain Name for each sending domain and select the Linked Tracking Domain you want to associate with it.
 1. Click the Add Domain or _Update Tracking Domain_ button to complete the association.
+Please note in order to associate a tracking domain with a sending doamin you need to make you have verified the trakcing domain first.
```
Comment (ewandennis) [inline articles/tech-resources/enabling-multiple-custom-tracking-domains.md]: - Lets separate your additions from the list item above. Add a blank line above your text to do that.
 - 2 typos: `trakcing` -> `tracking`.
- Missing word: `make you` -> `make sure you`
- Please link to the tracking domain part of the app in your text. Its this page: https://app.sparkpost.com/account/tracking-domains
 - How about `...on the right hand side of your tracking domain`.

(I recommend using the "View" button on your pull request ([this page](https://github.com/SparkPost/support-docs/pull/187/files)) so you can see how your changes look.)

**Comment by azikh** [comment]
Hi Ewan,

Thanks for your response. Im not sure how can I edited the article based on
your feedback.

Regards,
Azi

File: articles/tech-resources/enabling-multiple-custom-tracking-domains.md (hunk 1 above)
Comment (bkemper) [inline articles/tech-resources/enabling-multiple-custom-tracking-domains.md]: I don't want this note to get lost.  Can we move it before the first step?

## Associate your Tracking domain(s) with one or more Sending domains

To associate custom tracking domains to sending domains through the SparkPost user interface, the custom tracking domains must be verified first and then follow the steps below.

1. Select Account and then Sending Domains from the main menu.

File: articles/tech-resources/enabling-multiple-custom-tracking-domains.md [hunk 2]
```diff
@@ -47,6 +47,8 @@ To configure one or more custom tracking domains through the SparkPost user inte
     ![](media/enabling-multiple-custom-tracking-domains/xU06kJ2lac6YGEIi8KlReq8OaF1kfX.png)
 1. Complete the Sending Domain Name for each sending domain and select the Linked Tracking Domain you want to associate with it.
 1. Click the Add Domain or _Update Tracking Domain_ button to complete the association.
+Please note in order to associate a tracking domain with a sending doamin you need to make you have verified the trakcing domain first. 
+If you want to remove a trakcing domain you can simply go to the tracking domain section in your dashboard and click on the trash icon on the right hand side.
```
Comment (bkemper) [inline articles/tech-resources/enabling-multiple-custom-tracking-domains.md]: Can we combined this note with the other notes at the bottom?

4. Click the Add Domain or Update Tracking Domain button to complete the association. 

**Note:** For information on how to configure your Sending Domains, please see this article.  

**Note:** Any newly configured tracking domains have to route back, either through delegation or use of CNAME records, in order for us to be able to report on opens and clicks. Misconfiguration of your tracking domain(s) could mean link redirection could fail, so please ensure you test prior to placing any new domains into production!

**Note:** Deleting a tracking domain will automatically disassociate it from all sending domains.

**Pull request merged**